static/app/views/performance/database/useAvailableDurationAggregates.tsx

```diff
@@ -1,50 +1,26 @@
-import {decodeScalar} from 'sentry/utils/queryString';
-import {useLocation} from 'sentry/utils/useLocation';
 import useOrganization from 'sentry/utils/useOrganization';
 import {DEFAULT_DURATION_AGGREGATE} from 'sentry/views/performance/database/settings';
 
 // TODO: Type more strictly, these should be limited to only valid aggregate
 // functions
-type Query = {
-  aggregate: string;
-};
-
-type Result = {
-  availableAggregates: string[];
-  selectedAggregate: string;
-};
+type Result = string[];
 
 export function useAvailableDurationAggregates(): Result {
   const organization = useOrganization();
-  const location = useLocation<Query>();
 
-  let availableAggregates = ['avg'];
+  let availableAggregates: string[] = [];
```

Comment:
l: What do you think about `const` this and doing `availableAggregates.push(...['p50', 'p75', 'p95', 'p99'])` below?

Reply:
Works for me, but out of curiosity what's the advantage? I always feel a little icky about mutating a `const` even though it's allowed

Reply:
I think it's easier to read - `const` feels like a default, so when I see `let` it makes me pay more attention to how the variable is being mutated. This consideration gets more important with bigger function bodies.

Just `l` though, so I'm fine if we keep the existing way.

Reply:
Gotcha! I'll make the change, I see your point